hack/generate-etcd-secret.sh

```diff
@@ -4,17 +4,18 @@ set -x
 # only exit with zero if all commands of the pipeline exit successfully
 set -o pipefail
 
-IP=${1}
+NAMESPACE=openshift-kube-apiserver
+APISERVERPODNAME="$(kubectl -n ${NAMESPACE} get pod -lapp=openshift-kube-apiserver -ojson | jq -r '.items[0].metadata.name')"
```

Comment:
small nit to reduce the dependency on `jq`:
```
APISERVERPODNAME="$(kubectl -n ${NAMESPACE} get pod -lapp=openshift-kube-apiserver -o jsonpath='{.items[0].metadata.name}')
```

Reply:
done